recipes/regtools/run_test.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#help output is to stderr
+HELP="$(regtools --help 2>&1 >/dev/null)"
+VERSION="$(regtools --version 2>&1 >/dev/null)"
```

Comment:
Oh that is actually nice. But really not needed. We are happy if --version or --help is returning something that does not look like an error. Sorry. Can you just add --version and --help and remove this file?